features/promotion/managing_catalog_promotions/validating_catalog_promotion.feature

```diff
@@ -0,0 +1,46 @@
+@managing_catalog_promotions
+Feature: Validating a catalog promotion
```

Comment:
I'm not convinced to extracting these scenarios from `creating` feature as the intention is the same IMO. But, if you want to extract that, it should have a proper name associated with creation:
```suggestion
Feature: Validating a catalog promotion creation
```
or
```suggestion
Feature: Validating a creation of catalog promotion
```